### PR TITLE
Added pooling layer support and test cases

### PR DESCRIPTION
**Pull request by tchenot**

### CODE REVIEW

File: ai8xize.py [hunk 1]
```diff
@@ -2626,7 +2626,9 @@ def main():
     while ll < layers:
         if input_channels[ll] <= 0:
             eprint(f'Must specify `in_channels` for layer {ll}.')
-        if operator[ll] != op.NONE:
+        if operator[ll] != op.NONE and quantization[ll] is not None:
+            if quantization[ll] is None:
```
Comment (aniktash) [inline ai8xize.py:2630]: is this 2nd if correct? quantization is not None per previous line if

File: onnxcp.py [hunk 2]
```diff
@@ -645,6 +685,7 @@ def load(
     save_perm = None
     transpose_list = []  # pylint: disable=unused-variable
```
Comment (aniktash) [inline onnxcp.py]: transpose_list is not used anywhere

**Review by aniktash — CHANGES_REQUESTED** [review]
a few minor comments

File: ai8xize.py (hunk 1 above)
Comment (tchenot) [reply ai8xize.py:2630]: I'll take a look at this. It does not look right.

**Review by aniktash — APPROVED** [review]

**Review by vicloginov — APPROVED** [review]
Looks good

**Comment by tchenot** [comment]
Merging pool layer support

**Comment by tchenot** [comment]
Added pooling layer

**Pull request merged**